funding/manager_test.go

```diff
@@ -723,7 +723,8 @@ func openChannel(t *testing.T, alice, bob *testNode, localFundingAmt,
 // fundChannel takes the funding process to the point where the funding
 // transaction is confirmed on-chain. Returns the funding tx.
 func fundChannel(t *testing.T, alice, bob *testNode, localFundingAmt,
-	pushAmt btcutil.Amount, subtractFees bool, numConfs uint32,
+	pushAmt btcutil.Amount, subtractFees, fundMax bool, fundUpToMaxAmt,
+	minFundAmt btcutil.Amount, numConfs uint32, //nolint:unparam
```

Comment:
`nolint:unparam` was in place for `numConfs` which isn't used in this function either in order to prevent the linter from flagging this. But it seems like the linter isn't flagging it anymore so I've removed the `unparam`.
I've also removed the `fundMax` flag.